WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListView.swift

```diff
@@ -1,73 +1,104 @@
+    }
+}
+
 struct CustomFieldsListView: View {
     @Environment(\.presentationMode) var presentationMode
     @ObservedObject private var viewModel: CustomFieldsListViewModel
 
     let isEditable: Bool
-    let onBackButtonTapped: () -> Void
 
     init(isEditable: Bool,
-         viewModel: CustomFieldsListViewModel,
-         onBackButtonTapped: @escaping () -> Void) {
+         viewModel: CustomFieldsListViewModel) {
         self.isEditable = isEditable
         self.viewModel = viewModel
-        self.onBackButtonTapped = onBackButtonTapped
     }
 
     var body: some View {
-        NavigationStack {
-            List(viewModel.combinedList) { customField in
-                if isEditable {
-                    NavigationLink(destination: CustomFieldEditorView(key: customField.key, value: customField.value, onSave: { updatedKey, updatedValue in
-                        viewModel.saveField(key: updatedKey, value: updatedValue, fieldId: customField.fieldId)
-                    })) {
-                        CustomFieldRow(isEditable: true,
-                                       title: customField.key,
-                                       content: customField.value.removedHTMLTags,
-                                       contentURL: nil)
-                    }
-                } else {
-                    CustomFieldRow(isEditable: false,
-                                   title: customField.key,
-                                   content: customField.value.removedHTMLTags,
-                                   contentURL: nil)
-                }
+        List(viewModel.combinedList) { customField in
+            Button(action: { viewModel.selectedCustomField = customField }) {
+                CustomFieldRow(isEditable: isEditable,
+                               title: customField.key,
+                               content: customField.value.removedHTMLTags,
+                               contentURL: nil)
             }
-            .listStyle(.plain)
-            .navigationTitle(Localization.title)
-            .navigationBarTitleDisplayMode(.inline)
-            .toolbar {
-                ToolbarItem(placement: .cancellationAction) {
-                    Button(action: {
-                        onBackButtonTapped()
-                        presentationMode.wrappedValue.dismiss()
-                    }, label: {
-                        Image(systemName: "chevron.backward")
-                            .headlineLinkStyle()
-                    })
-                }
-
-                ToolbarItem(placement: .navigationBarTrailing) {
-                    if isEditable {
-                        HStack {
-                            Button {
-                                // todo-13493: add save handling
-                            } label: {
-                                Text("Save") // todo-13493: set String to be translatable
-                            }
-                            .disabled(!viewModel.hasChanges)
```

Comment:
@hafizrahman just letting you know, this behavior is broken after this PR, the save button is always enabled now if you want to fix it before the review.

Reply:
Good catch, thanks! Added https://github.com/woocommerce/woocommerce-ios/pull/14175/commits/9533311156c9bdaaeb717176e664b6cbb093e1be